tokio-io/src/codec/encoder.rs

```diff
@@ -3,6 +3,9 @@ use bytes::BytesMut;
 
 /// Trait of helper objects to write out messages as bytes, for use with
 /// `FramedWrite`.
+// Note: We can't deprecate this trait, because the deprecation carries through to tokio-codec, and
+// there doesn't seem to be a way to un-deprecate the re-export.
+// #[deprecated(since = "0.1.7", note = "Moved to tokio-codec")]
```

Comment:
Same as above, please remove the commented out deprecation and add an empty comment line above "Note".